### PR TITLE
Create auth

### PR DESCRIPTION
**Pull request by zpawn**
Story https://app.shortcut.com/deskpro/story/98878/youtrack-auth

### CODE REVIEW

**Review by amDavid01 — CHANGES_REQUESTED** [review]
We need to write tests to see if this works.

**Comment by zpawn** [comment]
> We need to write tests to see if this works.

hey @iTweeno it's work, trust me )))

**Comment by zpawn** [comment]
> We need to write tests to see if this works.

hey @iTweeno it's work, trust me)))

I think that it's necessary to make an MVP as soon as possible, and then write tests

**Comment by amDavid01** [comment]
I trust it works, but this is part of the new quality process in Apps, where we need,

1. Evidence of Acceptance Criteria
2. Tests written for major functions of the feature (with suitable vendor API mocks)

File: src/pages/VerifySettings/__tests__/VerifySettings.test.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { render } from "../../../testing";
+import { VerifySettings } from "../VerifySettings";
+
+describe("VerifySettings Page", () => {
+  test("render", async () => {
+    const { getByText } = render(<VerifySettings />, { wrappers: { theme: true }});
+
+    await waitFor(() => {
+      const text = getByText(/Verify Settings/i);
+      expect(text).toBeInTheDocument();
```
Comment (amDavid01) [inline src/pages/VerifySettings/__tests__/VerifySettings.test.tsx]: Alright so, In this test you're only seeing if verify Settings is inside the page, not actually seeing if the implementation works fine. Some API mocks would be required in order to see if this implementation works.

**Review by amDavid01 — CHANGES_REQUESTED** [review]

**Review by amDavid01 — APPROVED** [review]

**Pull request merged**